Bulldozer/Utility/AddMethods.cs

```diff
@@ -326,6 +327,75 @@ public static AttributeQualifier AddAttributeQualifier( RockContext rockContext,
             return newGroup;
         }
 
+        /// <summary>
+        /// Builds out a Serving Group heirarchy based off the heirarchy of a specific group.
+        /// </summary>
+        /// <param name="rockContext">todo: describe rockContext parameter on AddGroup</param>
+        /// <param name="topLevelServingGroup">The the top level serving group to build the group struture under.</param>
+        /// <param name="nonServingParentGroup">The parent group to copy from and use for cloning its heirarchy.</param>
+        /// <param name="copyCampus">Should the campus of the nonServingParentGroup be copied to the new group structure?.</param>
+        /// <param name="creatorPersonAliasId">todo: describe creatorPersonAliasId parameter on AddGroup</param>
+        /// <returns></returns>
+        public static List<Group> BuildParentServingGroupHeirarchy( RockContext rockContext, Group topLevelServingGroup, Group nonServingParentGroup, bool copyCampus = false, int? creatorPersonAliasId = null )
+        {
+            var groupHeirarchyToCopy = GetGroupHeirarchyAscending( nonServingParentGroup );
```

Comment:
If possible can you search and replace all typos of `Heirarchy` and replace with correct spelling? `Hierarchy`